opflexagent/opflex_conn_track.py

```diff
@@ -45,9 +45,9 @@ def main():
                "-p %s.%s -t opflex-conn-track") % (
                    sys.argv[1], sys.argv[2], sys.argv[3])
     LOG.debug("conn_track command: %s" % command)
-    sh(command)
+    ret = sh(command)
```

Comment:
nit: might be more straightforward and clean to just do "return sh(command)" then?

Reply:
pushed update, addressing this comment.